src/component/registry/types.ts

```diff
@@ -160,3 +161,40 @@ export type Stroke = {
    */
   color: 'inherit' | 'none' | 'swimlane' | string;
 };
+
+type Font = {
+  /**
+   * Possible values are all HTML color names or HEX codes.
+   */
+  color?: string;
+
+  // TODO To uncomment when we implement the Opacity in global/background/font/stroke
+  //opacity?: Opacity;
+
+  /**
+   *  The type of the value is int (in px).
+   */
+  size?: number;
+
+  family?: string;
+
+  /**
+   *  Default: false
```

Comment:
**nitpick:** Use the JSDoc usual syntax to document defaults (already used in existing types)
```suggestion
   *  @default false
```